client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js

```diff
@@ -0,0 +1,16 @@
+angular
+    .module("App")
+    .controller("WorkingStatusCtrl", class WorkingStatusCtrl {
+        constructor ($scope, Products) {
+            this.$scope = $scope;
+            this.Products = Products;
+
+            this.worksDetails = [];
+        }
+
+        $onInit () {
+            this.Products.getWorks("dedicated_server", true, true).then((works) => {
```

Comment:
`return` statement + `.catch()` maybe?

Reply:
How do you trigger component refresh if the variable already contains a promise?
When the promise resolves, it changes `this.worksDetails` from a promise to the final value, but this does not trigger the Angular binding.

Reply:
@euhmeuh i don't understand your answer :(

We should discuss about that :)